document/src/vespa/document/fieldset/fieldsets.h

```diff
@@ -54,17 +54,19 @@ class FieldCollection : public FieldSet
     FieldCollection(const FieldCollection &);
     FieldCollection(FieldCollection&&) = default;
     ~FieldCollection() override;
+    FieldCollection& operator=(const FieldCollection&) = default;
+    FieldCollection& operator=(FieldCollection&&) = default;
```

Comment:
You should add noexcept, both here and on the move constructor.

Reply:
Done.